LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1595,7 +1595,8 @@ webkit.org/b/261356 [ Debug ] fast/mediastream/device-change-event-2.html [ Pass
 webkit.org/b/263396 css3/color/text.html [ Skip ]
 
 # WebGPU
-[ Release ] http/tests/webgpu/webgpu/api/validation/encoding/cmds/copyTextureToTexture.html
+[ Release ] http/tests/webgpu/webgpu/api/validation/encoding/cmds [ Pass ]
+[ Release ] http/tests/webgpu/webgpu/api/validation/encoding/cmds/render/draw.html [ Skip ]
```

Comment:
Maybe we can't enable these for now, we see:
```
  http/tests/webgpu/webgpu/api/validation/encoding/cmds/copyTextureToTexture.html [ Pass Failure ]
  http/tests/webgpu/webgpu/api/validation/encoding/cmds/render/dynamic_state.html [ Pass Failure ]
  http/tests/webgpu/webgpu/api/validation/encoding/cmds/render/indirect_draw.html [ Pass Failure ]
  http/tests/webgpu/webgpu/api/validation/encoding/cmds/render/setVertexBuffer.html [ Pass Failure ]
  http/tests/webgpu/webgpu/api/validation/encoding/cmds/render/state_tracking.html [ Pass Failure ]
```
which is unrelated to your source changes